app/views/admins/contacts/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<% provide(:page_title, 'Detalhes do Contato') %>
+
+<div class="card">
+  <div class="card-header">
+    <h3 class="card-title"><%= @contact.name %></h3>
+    <div class="card-options">
```

Comment:
Faltou um botão voltar

Colocar uma legenda, pode ser um tooltip ou um título para cada informação
ex:
(fe fe-at-sign) Email: email@email.com

Trocar ícone  do whatsapp por um ícone de telefone, pode ser um número fixo ou até mesmo um número sem whats.

Reply:
Como o Pedro comentou, faltou um :title=> "Leganda" nos ícones.
Aqui está dando erro para carregar um arquivo de fonte feather-webfont.woff" (Failed to load resource: the server responded with a status of 404 (Not Found)) por conta disso as informações do contato e o botão editar contato estão com uma fonte padrão.

Reply:
> Como o Pedro comentou, faltou um :title=> "Legenda" nos ícones.
> Aqui está dando erro para carregar um arquivo de fonte feather-webfont.woff" (Failed to load resource: the server responded with a status of 404 (Not Found)) por conta disso as informações do contato e o botão editar contato estão com uma fonte padrão.

eita... Não foi adicionado nenhuma fonte, e apenas utilizamos as que estão no template... poderia me enviar um print deste erro para entendermos melhor?

> Trocar ícone do whatsapp por um ícone de telefone, pode ser um número fixo ou até mesmo um número sem whats.

Este era o nome do atributo Whatsapp não um telefone...

Reply:
> Faltou um botão voltar
Adicionado!

> 
> Colocar uma legenda, pode ser um tooltip ou um título para cada informação
> ex:
> (fe fe-at-sign) Email: [email@email.com](mailto:email@email.com)
> 
> Trocar ícone do whatsapp por um ícone de telefone, pode ser um número fixo ou até mesmo um número sem whats.

Reply:
Atualizei hoje a branch e mesmo assim apresentou o problema da fonte, segue as prints.
![Screenshot from 2019-04-29 13-00-22](https://user-images.githubusercontent.com/19822650/56909732-3428f100-6a7f-11e9-8801-a9184c9eade2.png)
![Screenshot from 2019-04-29 12-57-56](https://user-images.githubusercontent.com/19822650/56909733-3428f100-6a7f-11e9-9db8-eba36e8c46ac.png)
![Screenshot from 2019-04-29 12-57-43](https://user-images.githubusercontent.com/19822650/56909734-3428f100-6a7f-11e9-9528-0eb2c2359ebb.png)

Reply:
As prints sairam antes do pull, mas mesmo depois apresentou a mesma situação...